pkg/scheduler/framework/plugins/volumebinding/volume_binding_test.go

```diff
@@ -890,3 +891,90 @@ func TestVolumeBinding(t *testing.T) {
 		})
 	}
 }
+
+func TestIsSchedulableAfterCSIStorageCapacityChange(t *testing.T) {
+	table := []struct {
+		name   string
+		pod    *v1.Pod
+		oldCap interface{}
+		newCap interface{}
+		err    bool
+		expect framework.QueueingHint
+	}{
+		{
+			name: "pod has no pvc or generic ephemeral volumes",
+			pod:  makePod("pod-a").withEmptyDirVolume().Pod,
+			oldCap: &storagev1.CSIStorageCapacity{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "cap-a",
+				},
+				StorageClassName: "sc-a",
+			},
+			newCap: &storagev1.CSIStorageCapacity{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "cap-a",
+				},
+				StorageClassName: "sc-a",
+			},
+			err:    false,
+			expect: framework.QueueSkip,
+		},
+		{
+			name: "pod has one or more pvcs",
+			pod:  makePod("pod-a").withPVCVolume("pvc-a", "").Pod,
+			oldCap: &storagev1.CSIStorageCapacity{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "cap-a",
+				},
+				StorageClassName: "sc-a",
+			},
+			newCap: &storagev1.CSIStorageCapacity{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "cap-a",
+				},
+				StorageClassName: "sc-b",
```

Comment:
Are there any cases where the StorageClassName of CSIStorageCapacity is modified after its creation? If not, should we update the test case to reflect that? Same for L953.

```suggestion
				StorageClassName: "sc-a",
```